packages/studio-plugin/src/git/reloadGitData.ts

```diff
@@ -14,4 +15,11 @@ export default async function reloadGitData(
     throw new Error("Expected a GitData module.");
   }
   server.moduleGraph.invalidateModule(gitDataModule);
```

Comment:
invalidateModule was here to get hot reloading for free. I think we can delete it now

Reply:
I notice we still do the invalidation for studio data too even though we also send our own custom HMR update.  Should we remove it there as well?

Reply:
actually I think we still might need it for when you refresh the page, not sure